Automattic/crowdsignal-ui
# Add storybook

This patch adds support for storybook inside the repo.  
By default it's set up to pick up any `*.js` files inside a `/stories` directory.

Depends on #8.

# Testing

- Run `yarn install`
- Run `yarn storybook`
- Expect a new tab with our storybook to open.
